docs/reset-minikube.md

```diff
@@ -18,3 +18,25 @@ It is possible to get minikube confused such that it won't function properly. If
 1. See details about minikube as it starts
    * `--alsologtostderr -v=8  //shows log and errors for minikube start`
 1. The --cpus and --memory switches appear to be ingored on Windows, you must directly change the VM settings via Hyper-V Manager instead
+
```

Comment:
Should this go here, or in the `prerequisites-mac.md` file?

Reply:
I'll move it to prerequisites-mac.md and issue a different PR.

Reply:
Completed.